types/src/front/lib/core_api.ts

```diff
@@ -96,18 +96,19 @@ type GetDatasetsResponse = {
   datasets: { [key: string]: CoreAPIDatasetVersion[] };
 };
 
-export type CoreAPITableSchema = Record<
-  string,
-  "int" | "float" | "text" | "bool" | "datetime"
->;
+export type CoreAPITableSchema = {
+  name: string;
+  value_type: "int" | "float" | "text" | "bool" | "datetime";
```

Comment:
@fontanierh are these still up to date (I'm looking at datetime)

Reply:
Yes datetime is still supported